src/main/java/org/broadinstitute/hellbender/utils/genotyper/UnfilledReadsLikelihoods.java

```diff
@@ -1,27 +1,21 @@
 package org.broadinstitute.hellbender.utils.genotyper;
 
-import com.google.common.annotations.VisibleForTesting;
 import htsjdk.samtools.util.Locatable;
 import htsjdk.variant.variantcontext.Allele;
-import it.unimi.dsi.fastutil.objects.Object2IntMap;
-import org.broadinstitute.hellbender.utils.IndexRange;
 import org.broadinstitute.hellbender.utils.SimpleInterval;
-import org.broadinstitute.hellbender.utils.Utils;
 import org.broadinstitute.hellbender.utils.pileup.PileupElement;
 import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.function.ToDoubleFunction;
 
 /**
  * Dummy Read-Likelihood container that computes partial likelihoods based on the read pileups for snps.
  *
  * Note: this class uses FastUtil collections for speed.
  */
-public class UnfilledReadsLikelihoods<A extends Allele> extends ReadLikelihoods<A> {
+public class UnfilledReadsLikelihoods<A extends Allele> extends AlleleLikelihoods<GATKRead, A> {
```

Comment:
What does "Unfilled" mean ... can you include that in the javadoc?

Reply:
done

Reply:
Ok now I see, I have to say that this class smell a bit... so basically is a class for Likelihood tables that actually cotain "garbage" likelihoods and are only used as a a list of evidences or alleles.... Wouldn't make more sense to change the code that use the likelihood table in that way to simply accept a list of reads or samples or both? 

I don't see the point of such a class you could simple create a likelihood table that you never care to fill with meaninful likelihoods. 

Or am I missing something here?